### PR TITLE
Use password inputs for password fields

### PR DESCRIPTION
**Pull request by galfert**
I noticed that the space config forms used normal inputs for the password fields.

### CODE REVIEW

**Comment by skddc** [comment]
Anyone has an idea about that Chrome error on Travis?

**Comment by skddc** [comment]
Can we just go back to good old phantomjs? The whole new Chrome setup is creating problems everywhere we use it. :/

**Comment by galfert** [comment]
The tests on Travis are all green now.

File: app/components/space-setting-modal/template.hbs [hunk 1]
```diff
@@ -21,14 +21,11 @@
...
   <p>
-    {{input value=space.server.password placeholder="password"}}
+    {{input type="password" value=space.server.password placeholder="password"}}
   </p>
   <p>
     {{input value=space.server.nickname required="required" placeholder="nickname"}}
   </p>
-  <p>
-    username:  {{space.server.username}}
-  </p>
```
Comment (skddc) [inline app/components/space-setting-modal/template.hbs]: This is removing the username?

Comment (galfert) [reply app/components/space-setting-modal/template.hbs]: No, this was outputting the username below the form from the actual username input field. Don't know why this was there in the first place, I guess it was for debugging purposes.

Comment (skddc) [reply app/components/space-setting-modal/template.hbs]: Ah, ok.

**Pull request merged**